### PR TITLE
Add and fix links to non-RHEL images

### PR DESCRIPTION
**Pull request by hhorak**

### CODE REVIEW

**Comment by torsava** [comment]
Overall +1, just one note: let's mention it's running on UBI micro specifically?

**Comment by hhorak** [comment]
Good point, will add micro note..

File: 3.9-minimal/README.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ provided as a [S2I](https://github.com/openshift/source-to-image) base image for
 Because the minimal and full images work similarly, we document here only the differences and limitations
 of the minimal container image. For the documentation of common features see the [full container image docs](https://github.com/sclorg/s2i-python-container/tree/master/3.9).
 
+The Python 3.9 minimal container image is currently considered a tech-preview and only available in the quay.io.
```
Comment (hroncok) [inline 3.9-minimal/README.md:10]:
```suggestion
The Python 3.9 minimal container image is currently considered a tech-preview and only available on quay.io.
```

**Review by frenzymadness — APPROVED** [review]
I like it, thank you. Feel free to merge it if you have no more updates in the queue.

**Comment by hhorak** [comment]
All should be addressed, so merging.

**Pull request merged**